aws_neuron/assets/dashboards/aws_neuron_overview.json

```diff
@@ -0,0 +1,1191 @@
+                "avg",
+                "min",
+                "max",
+                "value",
+                "sum"
+              ],
+              "type": "timeseries",
+              "requests": [
+                {
+                  "formulas": [
+                    {
+                      "alias": "Memory usage",
+                      "number_format": {
+                        "unit": {
+                          "type": "canonical_unit",
+                          "unit_name": "percent"
+                        }
+                      },
+                      "formula": "100 * query2 / query3"
+                    },
+                    {
+                      "alias": "vCPU usage",
+                      "number_format": {
+                        "unit": {
+                          "type": "canonical_unit",
+                          "unit_name": "percent"
+                        }
+                      },
+                      "formula": "100 * query4"
+                    },
+                    {
+                      "alias": "Neuroncore usage",
+                      "number_format": {
+                        "unit": {
+                          "type": "canonical_unit",
+                          "unit_name": "percent"
+                        }
+                      },
+                      "formula": "100 * query6"
+                    }
+                  ],
+                  "queries": [
+                    {
+                      "data_source": "metrics",
+                      "name": "query2",
+                      "query": "avg:aws_neuron.system.memory.used_bytes{$instance_id, $instance_type, $availability_zone, $region, $subnet_id}"
+                    },
+                    {
+                      "data_source": "metrics",
+                      "name": "query3",
+                      "query": "avg:aws_neuron.system.memory.total_bytes{$instance_id, $instance_type, $availability_zone, $region, $subnet_id}"
+                    },
+                    {
+                      "data_source": "metrics",
+                      "name": "query4",
+                      "query": "avg:aws_neuron.system.vcpu.usage_ratio{$instance_id, $instance_type, $availability_zone, $region, $subnet_id}"
+                    },
+                    {
+                      "data_source": "metrics",
+                      "name": "query6",
+                      "query": "avg:aws_neuron.neuroncore.utilization_ratio{$instance_id, $instance_type, $availability_zone, $region, $subnet_id}"
+                    }
+                  ],
+                  "response_format": "timeseries",
+                  "style": {
+                    "palette": "datadog16",
+                    "order_by": "values",
+                    "color_order": "shuffled",
+                    "line_type": "solid",
+                    "line_width": "normal"
+                  },
+                  "display_type": "area"
```

Comment:
I don't think area would be the best representation here. I think areas stack on top of each other. So for example if 2 unrelated things are showing 100% then on the graph you see if up to 200%